modules/storage-expanding-add-volume-expansion.adoc

```diff
@@ -12,7 +12,16 @@ have the `allowVolumeExpansion` field set to `true`.
 
 .Procedure
 
-* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute.
+* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute by using the following command:
++
+[source,terminal]
+----
+$ oc edit storageclass <storage_class_name> \// <1> 
+   -n <storage_class_namespace> <2>
```

Comment:
Hi @cbippley Storageclass is a cluster-wide resource, so no need the "-n <storage_class_namespace>"